docs/04-api-reference/front-commerce-core/services.mdx

```diff
@@ -21,7 +21,7 @@ implementation.
 | Name                               | Description                                | Reference                                                                                                                                                                                                                                                                            |
 | ---------------------------------- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `ContentMetadataExtractorRegistry` | Used for storefront content registration   | [📖 Guide](/docs/3.x/guides/magic-button/x-ray#register-the-extractor-in-the-contentmetadataextractorregistry) <br/> [🔗 Code](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/core/services/implementations/ContentMetadataExtractorRegistry.ts) |
-| `ServerEventBus`                   | Used for server events publishing          | [📖 Guide](/docs/3.x/guides/server-side-events/create-an-event#use-the-event) <br/> [🔗 Code](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/core/services/implementations/ServerEventBus.ts)                                                    |
+| `ServerEventBus`                   | Used for server events publishing          | [📖 Guide](/docs/3.x/guides/server-side-events/how-to/create-an-event#use-the-event) <br/> [🔗 Code](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/core/services/implementations/ServerEventBus.ts)                                             |
```

Comment:
**question:** Shouldn't the guide link be a link to the whole guide instead of a specific section ? I understand that the `ServerEventBus` is only used in this section specifically, but this section alone doesn't really mean anything without the rest of the guide.

Reply:
Linked to the whole guide